app/Commands/Sites/CreateCommand.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Commands\Sites;
+
+use App\Commands\BaseCommand;
+use App\Commands\Concerns\HasServerIdParameter;
+use App\Helpers\OptionsHelper;
+use App\Questions\Ask;
+use App\Questions\Choice;
+use App\Questions\Confirm;
+use App\Questions\HasQuestions;
+use Illuminate\Support\Str;
+
+class CreateCommand extends BaseCommand
+{
+    use HasQuestions;
+    use HasServerIdParameter;
+
+    protected $signature = 'sites:create
+                            {installation_method : Type of installation (wp or blank)}
+                            {server_id? : Server ID}
+                            {--domain= : Domain name}
+                            {--site_user=}
+                            {--db_name=}
+                            {--db_user=}
+                            {--db_pass=}
+                            {--wp_title=}
+                            {--wp_admin_user=}
+                            {--wp_admin_email=}
+                            {--wp_admin_pass=}
+                            {--php_version=}
+                            {--page_cache_enabled}
+                            {--https_enabled}
+                            {--profile=}
+                            {--f|force}';
+
+    protected $description = 'Create a site';
+
+    protected array $userInput;
+
+    protected function action(): int
+    {
+        if (!in_array($this->argument('installation_method'), OptionsHelper::INSTALLATION_METHODS, true)) {
+            $this->error('Invalid site type.');
+            $this->newLine(1);
+            return self::INVALID;
+        }
+
+        $server = $this->selectServer('deploy to')->first();
+
+        if (is_null($server)) {
+            return self::INVALID;
+        }
+
+        $this->userInput['domain'] = Ask::make('Domain')
+            ->nonInteractive($this->nonInteractive())
+            ->resolveAnswer($this);
+
+        $this->userInput += $this->askQuestions($this->nonInteractive());
+
+        $site = $this->spinupwp->createSite($server->id, array_merge($this->arguments(), $this->options(), $this->userInput));
+
+        $this->successfulStep("{$site->domain} is {$site->status} (event_id = {$site->eventId()})");
+
+        return self::SUCCESS;
+    }
+
+    public function getDomainSlug(): string
+    {
+        return str_replace('.', '', $this->userInput['domain']);
+    }
+
+    public function questions(): array
+    {
+        $commonStart = [
+            Confirm::make('Https Enabled')
+                ->withDefault((bool) !$this->nonInteractive()),
+
+            Ask::make('Site User')
+                ->withDefault($this->getDomainSlug()),
+        ];
+
+        $db = [
+            Ask::make('Db Name')
+            ->withDefault($this->getDomainSlug()),
+
+            Ask::make('Db User')
+                ->withDefault($this->getDomainSlug()),
+
+            Ask::make('Db Pass')
+                ->withDefault(Str::random(12)),
```

Comment:
Let's show the default password as (auto-generated) and display the actual password in the success output.

Reply:
✅

Reply:
> Let's show the default password as (auto-generated)

![CleanShot 2022-03-01 at 13 02 43@2x](https://user-images.githubusercontent.com/1422996/156173763-3b24c6cf-286b-4e30-9642-f22eaf8c4878.png)

Reply:
@A5hleyRich if you're _not_ seeing these defaults with the current build, then this is a bug that I can't reproduce. Any steps to reproduce?

Reply:
@ohryan I was referring to that "auto-generated" should be shown as the default text, not the actual generated password. I don't feel strongly about this now that we're displaying the passwords in the output so ignore this. We can always iterate in the future.